### PR TITLE
setup.cfg: black and flake8 agree on error W504

### PR DESCRIPTION
**Pull request by cclauss**
Failing test fixed in #529

### CODE REVIEW

**Comment by cclauss** [comment]
https://black.readthedocs.io/en/stable/contributing/issue_triage.html?highlight=flake8#common-reports

File: setup.cfg [hunk 1]
```diff
@@ -6,8 +6,9 @@ profile = black
 known_third_party=six,coverage,mock
 
 [flake8]  # black and isort compatible rules
+max-complexity = 10
 max-line-length = 88
-ignore = W503,W504,E203
+ignore = E203
```
[the remark this answers is not in the document]
Comment (cclauss) [reply setup.cfg:11]: Let's wait until we drop legacy Python and can upgrade to the current version of `black`.

[the remark this answers is not in the document]
Comment (hugovk) [reply setup.cfg:11]: What's the difference in E203 handling between Black 21.12b0 and current? I think it's been the same for a long time.

[the remark this answers is not in the document]
Comment (cclauss) [reply setup.cfg]: Now I get it.  You are right.  Your first review made me think there was a change made to make black and flake8 agree more.

**Comment by cclauss** [comment]
https://black.readthedocs.io/en/stable/faq.html?highlight=W504#why-are-flake8-s-e203-and-w503-violated

**Comment by sirosen** [comment]
Apologies if I'm missing it in the messages here, but extend-ignore is the current recommendation from `black`, so I'd like to go with that: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8

It looks like that's what's currently in this PR, so that's great!

However, this is also setting `max-complexity`, which is a lint that is just slipped in here and wasn't in use before. I might -- probably would be! -- open to adding additional checks. But I would prefer that we keep them either separate, or discussed beforehand.

Rather than blocking on that, I've pushed a commit to comment out the max-complexity setting. As soon as CI passes, I intend to squash-merge.

**Pull request merged**